tests/validators/test_model_init.py

```diff
@@ -41,7 +46,7 @@ def test_model_init():
 def test_model_init_nested():
     class MyModel:
         # this is not required, but it avoids `__pydantic_fields_set__` being included in `__dict__`
-        __slots__ = '__dict__', '__pydantic_fields_set__', '__pydantic_extra__', '__pydantic_private__'
+        __slots__ = ('__dict__', '__pydantic_fields_set__', '__pydantic_extra__', '__pydantic_private__')
```

Comment:
I personally prefer having the parenthesis so I'm +1 on this, but I'm curious if this was a linter or your preference as well?

Reply:
Ah, with apologies this was black, for which the vscode configuration temporarily glitched out for me earlier. I'll revert as unrelated to this PR.